event-handler/cli.go

```diff
@@ -18,12 +18,15 @@ package main
 
 import (
 	"context"
+	"strings"
 	"time"
 
 	"github.com/alecthomas/kong"
 
 	"github.com/gravitational/teleport-plugins/event-handler/lib"
 	"github.com/gravitational/teleport-plugins/lib/logger"
+	"github.com/gravitational/teleport-plugins/lib/stringset"
+	"github.com/gravitational/trace"
```

Comment:
nit: import grouping for external packages
```suggestion
	"github.com/gravitational/teleport-plugins/event-handler/lib"
	"github.com/gravitational/teleport-plugins/lib/logger"
	"github.com/gravitational/teleport-plugins/lib/stringset"

	"github.com/alecthomas/kong"
	"github.com/gravitational/trace"
```